rlberry/utils/check_bandit_agent.py

```diff
@@ -1,8 +1,8 @@
-from rlberry.envs.bandits import NormalBandit
+from rlberry.envs.bandits import BernoulliBandit
 from rlberry.manager import AgentManager
 
 
-def check_bandit_agent(Agent, environment=NormalBandit):
+def check_bandit_agent(Agent, environment=BernoulliBandit, seed=None):
```

Comment:
the seed has to be fixed. It can be any int, but it can't be None because if it is None, there would be reproducibility issues. Maybe you can change the default to some int (42 is a good choice :) )

Reply:
Agreed, done :)